packages/media/EmbedVideo.jsx

```diff
@@ -1,11 +1,13 @@
 import React from "react";
 
-export default function EmbedVideo(videoId) {
+export default function EmbedVideo(props) {
+  const { videoId, className } = props;
   return (
```

Comment:
@hmalcolmSquiz FYI, I have updated this Embed Video package to add a `className` prop so we can add classes to the `<iframe>` when necessary – I've done this in the `video-hero` in order to make the YouTube embed responsive. Please take a look and let me know if you think this is going to cause any problems for existing things. I've checked a bit and everything seems to be fine. Thanks!